pkg/apps/application.jsx

```diff
@@ -80,11 +80,11 @@ class Application extends React.Component {
 
             return description.map(paragraph => {
                 if (paragraph.tag == 'ul') {
-                    return <ul>{paragraph.items.map(item => <li>{item}</li>)}</ul>;
+                    return <ul key={paragraph}>{paragraph.items.map(item => <li key={item}>{item}</li>)}</ul>;
                 } else if (paragraph.tag == 'ol') {
-                    return <ol>{paragraph.items.map(item => <li>{item}</li>)}</ol>;
+                    return <ol key={paragraph}>{paragraph.items.map(item => <li key={item}>{item}</li>)}</ol>;
```

Comment:
Key is not a string.

Reply:
fixed via indexes. Or anything better (natural key)?